### ISSUE TITLE
IE11 webpack export not working

### ISSUE THREAD

**Issue by grasmachien** [amcharts/amcharts4#933]
I have installed amcharts with NPM and then I import everything like this.
<img width="368" alt="screenshot 2019-02-06 at 17 15 19" src="https://user-images.githubusercontent.com/6026424/52355563-d01fff00-2a32-11e9-823b-1a320b9d9515.png">

```
import * as am4core from '@amcharts/amcharts4/core'
import * as am4maps from '@amcharts/amcharts4/maps'
import am4geodata_worldLow from '@amcharts/amcharts4-geodata/worldLow'
```

This seems to create chunks for the vendors.
<img width="829" alt="screenshot 2019-02-06 at 16 33 01" src="https://user-images.githubusercontent.com/6026424/52355185-28a2cc80-2a32-11e9-9b43-172e719560f7.png">

I have also included and installed the shims for older browsers at the top of the JS code.
`import 'core-js/shim'`

And yet the export to jpg and png still fails...
![chunks](https://user-images.githubusercontent.com/6026424/52355284-5d168880-2a32-11e9-8348-3570d9b51349.png)

I have previously tried inserting the Amcharts libs in script tags aboven my main.js (which is bundled with webpack) script tag.
This also gave me chunk errors.
![ie11export 1](https://user-images.githubusercontent.com/6026424/52355456-a4047e00-2a32-11e9-9aac-d6010f69d3d8.jpg)

The way I see this at the moment is that IE11 support is completely broken in this release.
At least for exporting.

Thanks for looking at this.

**Comment by grasmachien** [comment]
I have built an example to reproduce the problem.
You can download the ZIP and install with npm install.
Building can be done with npm run prepare.

[AMCHARTS_WEBPACK.zip](https://github.com/amcharts/amcharts4/files/2840998/AMCHARTS_WEBPACK.zip)

This is the error you will get when exporting to anything other than SVG in IE11.
![error](https://user-images.githubusercontent.com/6026424/52414663-382f1d80-2ae5-11e9-8ed2-e8b6bf5fb304.png)

**Comment by jimmykane** [comment]
I think this relates to this one https://github.com/webpack/webpack/issues/8656 

@martynasma  I am experiencing the same with webpack 4.29.0 

@grasmachien can you try downgrading webpack to 4.28.0 ?

**Comment by grasmachien** [comment]
@jimmykane Downgrading from 4.28.4 to 4.28.0 did not fix it.
In the actual project I'm running webpack 4.16.5 so this probably is not related.

**Comment by grasmachien** [comment]
@jimmykane Were you able to produce a working example with webpack and IE11 ?

**Comment by Pauan** [comment]
@grasmachien Thanks for the report! I've been able to reproduce it. We're investigating how to fix this.

**Comment by Pauan** [comment]
@grasmachien We found and fixed the bug. It will be in the next release. We'll let you know when it's released.

Also, for importing amCharts you shouldn't be using both `<script>` tags and Webpack, only one or the other. So you should remove the `<script>` tags which import amCharts.

**Comment by martynasma** [comment]
Fixed in [4.1.8](https://github.com/amcharts/amcharts4/releases/tag/4.1.8).

## [4.1.8] - 2019-02-15

### Added
- JSON: You can now specify both list of items and template settings for `ListTemplate`, e.g.: `titles: { template: { ... }, values: [{ ... }] }`. Specifying it the old way will still work.

### Changed
- `DateAxis` will now pay attention to `dateFormatter.firstDayOfWeek` when grid is in "weekly" mode.

### Fixed
- Using `proprtyFields` on a `LineSeries` sometimes could result in an error. (fix by @AndiLi99)
- French locale (fr_FR) updated for correct decimal/thousands separators.
- Export: `scale` in image export options was being ignored.
- JSON: `cursor.snapToSeries` was not working in JSON configs.
- Selection was acting funky when zooming `DateAxis` and cursor moved out of plot area.
- An infinite error message loop fixed which was happening if series was added and there was no X or Y axes defined.
- Tooltip on the last available data item was left visible even if cursor moved to the date where the series had no data points.
- When `snapToSeries` was enabled in `XYCursor`, vertical line was not shown.
- [Issue 933.](https://github.com/amcharts/amcharts4/issues/933)

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
